.github/workflows/install_deps/action.yml

```diff
@@ -95,11 +99,10 @@ runs:
         python setup.py bdist_wheel
         pip install dist/PennyLane*.whl
 
-    - name: Install Catalyst
+    - name: Install Catalyst from nightly build
       shell: bash
-      if: inputs.install_catalyst_after_pennylane == 'true'
-      # TODO: replace after release
-      run: pip install --upgrade pennylane-catalyst
+      if: inputs.install_catalyst_nightly == 'true'
```

Comment:
If this is false, do we want to install the stable release from pypi? Or not install catalyst at all?

Reply:
Not at all. The reason is that we only have one test job that installs `catalyst`. The purpose of adding this was basically future proofing. If we do update our current tests to install catalyst-stable, they can do that using the `additional_pip_packages` argument.